mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -445,11 +446,10 @@ case class ScalaPresentationCompiler(
       doc <- ParsedComment.docOf(sym)
     } yield doc
 
-    if (documentation.nonEmpty) {
-      item.setDocumentation(hoverContent(None, None, documentation))
```

Comment:
I may have missed the discussion, but why is this gone?

Reply:
@gabro _That_ looks like a typo on my part...Sorry about that.

Reply:
@gabro updated.